modules/ROOT/pages/understanding-editor-loads.adoc

```diff
@@ -0,0 +1,65 @@
+= Understanding editor loads
+:navtitle: Understanding editor loads
+:description: Relevant information for Tiny Cloud users to help understand editor loads for {productname}.
+:keywords: invalid-api-key, API, {productname}, cloud, frequently asked questions
+
+== Understanding editor loads for {productname}
+
+[IMPORTANT]
+This information is only relevant to Tiny Cloud users. Users who self-host the open source version of {productname} are not subject to editor load restrictions, but must comply with the ////<Insert link>[open source license].//// 
+
+An editor load is the event that occurs each time {productname} is initialized in your application. The editor dispatches the 'init' event to indicate a successful load. For example, if 100 users load {productname} 10 times each, the result would be 1,000 editor loads. 
+
+The process of initializing an editor involves several steps.
+
+. The integrator initializes one or more editors on the webpage, commonly accomplished through the `tinymce.init({ ... })` method. 
++
+Regardless of the method used, the following pattern is adhered to when adding an editor to a webpage:
++
+.. Create a new editor instance object.
+.. Add the editor instance to the editor manager.
+.. Call `editor.render()`.
+
+. Calling `editor.render()` triggers a sequence of functions that prepare the editor for user interaction. Notably:
++
+.. All required scripts, including theme, model, language, icons, skin, and plugins, are loaded and initialized.
+.. The editor UI is rendered.
+.. The target element on the webpage is transformed into an editor instance.
+.. Content CSS is loaded onto the page.
+.. Initial content is loaded into the editor from the target element.
+
+. If all these steps are successful, the final initialization steps are completed, including:
++
+.. Setting the `editor.initialized` property to `true`.
+.. Dispatching the `init` event.
+.. Marking the editor as the active editor in the editor manager.
+.. Setting the selection to a valid position in the editor and making it focused so the user can interact with it.
```

Comment:
This looks like it came from our training material? Sorry, but I think it's unnecessary and will only confuse developers. They don't need to understand the entire load process, just why they're creating more editor loads than they expected to.

This suggestion is not valid adoc syntax, but here's the general idea I'm looking for:
```suggestion
The `selector` property of the `init` method determines which elements should be replaced with editor instances. For example if the selector is `textarea` and there are 5 textarea elements on the page, 5 editor instances will be created.

Each editor instance performs the same initialization sequence and then dispatches an `init` event. This event is dispatched even if the the editor is not visible.
```